tema9Ianuarie/src/com/sci/bv/tema9Ianuarie/MainTest.java

```diff
@@ -0,0 +1,14 @@
+package com.sci.bv.tema9Ianuarie;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class MainTest {
+    Main Max = new Main();
+
+    @Test
+    void getMax() {
+        Assertions.assertEquals(Max.getMax(5, 10), 10);
```

Comment:
this is one test, I would also test 10 and 5 and also 10 and 10. So I can be sure it works for all options